charts/seed-operatingsystemconfig/original/templates/kubelet/_kubelet-monitor.service

```diff
@@ -13,8 +13,10 @@
     EnvironmentFile=/etc/environment
     {{- if semverCompare "< 1.17" .Values.kubernetes.version }}
     ExecStartPre=/usr/bin/docker run --rm -v /opt/bin:/opt/bin:rw {{ required "images.hyperkube is required" .Values.images.hyperkube }} /bin/sh -c "cp /usr/local/bin/kubectl /opt/bin"
-    {{- else }}
+    {{- else if semverCompare "< 1.19" .Values.kubernetes.version }}
     ExecStartPre=/usr/bin/docker run --rm -v /opt/bin:/opt/bin:rw --entrypoint /bin/sh {{ required "images.hyperkube is required" .Values.images.hyperkube }} -c "cp /usr/local/bin/kubectl /opt/bin"
+    {{- else }}
+    ExecStartPre=sh -c "ID=\"$(/usr/bin/docker run --rm -d -v /opt/bin:/opt/bin:rw {{ required "images.hyperkube is required" .Values.images.hyperkube }})\"; /usr/bin/docker cp \"$ID\":/kubectl /opt/bin; docker stop \"$ID\"; chmod +x /opt/bin/kubectl"
```

Comment:
I tried this PR also with suse-chost images and machines were failing to join. 

```
# systemctl status kubelet
● kubelet.service - kubelet daemon
   Loaded: error (Reason: Exec format error)
   Active: inactive (dead)
     Docs: https://kubernetes.io/docs/admin/kubelet

# journalctl -u kubelet

Sep 01 11:55:54 ip-10-222-0-12 systemd[1]: /etc/systemd/system/kubelet.service:15: Executable path is not absolute: sh -c "ID=\"$(/usr/bin/docker run --rm -d -v /opt/bin:/opt/bin:rw eu.gcr.io/gardener-project/hyperkube:v1.19.0)\"; /usr/bin/docker cp \"$ID\":/kubelet /opt/bin; docker stop \"$ID\"; chmod +x /opt/bin/kubelet"
```

Reply:
I changed it to `/usr/bin/env sh`, thanks. Can you confirm it works now?

Reply:
Yep, works now.